Specs/DateTime/Arabic/DatePeriodParser.json

```diff
@@ -3600,10 +3551,9 @@
   {
     "Input": "سيحدث بعد يومين.",
     "Context": {
-      "ReferenceDateTime": "2016-11-07T00:00:00"
+      "ReferenceDateTime": "2017-11-08T00:00:00"
```

Comment:
Why was it necessary to change the reference time?

Reply:
it seems it was unintentionally changed, restored the original.